resource/doc/BuildForWindows.md

```diff
@@ -1,8 +1,19 @@
-# Prepare  [![AppVeyor Build Status](https://ci.appveyor.com/api/projects/status/68i57b8ssasttngg?svg=true)](https://ci.appveyor.com/project/li-zhi/vmaf)
-  - Visual Studio 2015 on Windows
+# Building vmaf on Windows
+
+We are going to build vmaf in Windows. These steps are in accordance with the corresponding github action for building on Windows and have been tested successfully using a Windows10 machine. They work with either `cmd` or `PowerShell`.
+
+**Note:** This guide is just to build libvmaf on Windows and not involves the python part of project as it is the same across all platforms (settings up virtual environment, ...).
+
+## Steps
+  1.Install [msys2](https://www.msys2.org/)
```

Comment:
```suggestion
  1. Install [msys2](https://www.msys2.org/)
```

markdown requires a space between the number and the string afterward for it to be used as an `ol`

Reply:
sorry. fixed.